tests/tools.vitruv.applications.demo.familiespersons.tests/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,9 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=11
+org.eclipse.jdt.core.compiler.compliance=11
+org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
+org.eclipse.jdt.core.compiler.problem.enablePreviewFeatures=disabled
+org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
+org.eclipse.jdt.core.compiler.problem.reportPreviewFeatures=warning
+org.eclipse.jdt.core.compiler.release=enabled
+org.eclipse.jdt.core.compiler.source=11
```

Comment:
Where does this file come from? We don't have that currently in any other test project.

Reply:
This is a file automatically generated by Eclipse when setting up a project with a specific Java version. Since we use Java 11 (and current Eclipse versions ship with Java 17), we have to define the Java version if you do not want to manually define it everytime you set up a workspace (like with the UTF-8 encoding). I was not aware that we do not have it in other projects. So shall I remove it? Or shall we add it to other projects as well? We can, of course, at least remove the information not related to the Java version.

Reply:
Maybe it makes sense to leave them out in this PR (and the related ones; I spotted these files there too) and have a subsequent PR that adds the files to all projects? Otherwise, we will introduce inconsistencies. In a dedicated PR, we can also discuss whether we want to include some of the additional flags present here not related to the Java version.

Reply:
Yes, that makes sense. I habe removed them in d43f678 and check the related PRs as well.

Reply:
I have also updated the Vitruv-DSLs PR accordingly.